source/plugin/Assets/GoogleMobileAds/Platforms/Unity/BannerClient.cs

```diff
@@ -117,6 +116,7 @@ public void CreateBannerView(string adUnitId, AdSize adSize, int x, int y)
         public void LoadAd(AdRequest request)
```

Comment:
public void CreateBannerView(string adUnitId, AdSize adSize, int x, int y), there's no check for adaptive banner. Is that happening in a separate CL?